inference-engine/src/transformations/include/transformations/snippets/collapse_subgraph.hpp

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2018-2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
```

Comment:
I propose to use different folders for transformations and operations, something like:
```
snippets/
    transformations/
    op/
```

Reply:
Ok. with an addition, I prefer `pass`. I don't like `transformation` naming in general. `Pass` is shooter and close to domain terminology